README.md

```diff
@@ -1,11 +1,41 @@
-# Autoscale AWS Library
+# FortiGate Autoscale on AWS
 
-This is an intial readme template.
+The implementation for FortiGate Autoscale using [Fortinet Autoscale](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
+
+This project provides a multi-group Auto Scaling functionality for Fortinet FortiGate EC2 instances to form an HA cluster with failover protection.
+
+This project has the following features:
+
+1. Multi-group Hybrid Licensing models:
+   1. **Hybrid**: 1 fix-sized Auto Scaling group of 2 (and more) BYOL FortiGate Instances, and 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
+   2. **BYOL-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) BYOL FortiGate Instances.
+   3. **PAYG-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
+2. AWS Transit Gateway Integration.
+
+## Deployment packages
+
+  * Download the **aws-cloudformation.zip** for the latest production version from [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).
+  * or, manually generate the deployment package:
+    1. Chechkout the **main** branch of the project.
+    2. Run `npm run build-artifacts` at the project root directory.
+    3. Deployment packages **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
+
+## Deployment guides
+
+Deployment guides are available from the Fortinet Document Library:
+
+  + [<NAME_PLACE_HOLDER>](<LINK_PLACE_HOLDER>)
```

Comment:
@klc178 please update it here with the doc reference to the official documentation.

Reply:
```suggestion
  + [ FortiGate / FortiOS 6.2 Deploying auto scaling on AWS](https://docs.fortinet.com/vm/aws/fortigate/6.2/aws-cookbook/6.2.0/397979/deploying-auto-scaling-on-aws)
    + [ FortiGate / FortiOS 6.4 Deploying auto scaling on AWS](https://docs.fortinet.com/vm/aws/fortigate/6.4/aws-cookbook/6.4.0/397979/deploying-auto-scaling-on-aws)
```

Reply:
are we supporting 6.4? It isn't in the drop down.

Reply:
> are we supporting 6.4? It isn't in the drop down.

Not in this version but in the plan to support it soon.